tools/wpt/testfiles.py

```diff
@@ -46,12 +46,13 @@
 def get_git_cmd(repo_path):
     # type: (bytes) -> Callable[..., Text]
     """Create a function for invoking git commands as a subprocess."""
+    devnull = open(os.devnull, "w")
     def git(cmd, *args):
         # type: (Text, *Union[bytes, Text]) -> Text
         full_cmd = [u"git", cmd] + list(item.decode("utf8") if isinstance(item, bytes) else item for item in args)  # type: List[Text]
         try:
             logger.debug(" ".join(full_cmd))
-            return subprocess.check_output(full_cmd, cwd=repo_path, stderr=subprocess.STDOUT).decode("utf8").strip()
+            return subprocess.check_output(full_cmd, cwd=repo_path, stderr=devnull).decode("utf8").strip()
```

Comment:
Does it make more sense to just set it as `None` (i.e. the default) so it inherits from the parent and appears in the logs?

FWIW I think the original idea here was that some commands might print something meaningful to stderr indicating an actual failure. I don't really know how to audit that we aren't relying on that in a sane way.

Reply:
Changed to `None` (i.e. removed the kwarg). 

> some commands might print something meaningful to stderr indicating an actual failure

At least `git` shouldn't (and AFAIK, doesn't) do that, and this line is only used when calling `git`.